lib/actions/pandora.js

```diff
@@ -37,15 +37,15 @@ function pandora(player, values) {
         
                    return player.coordinator.setAVTransport(uri, metadata) 
                      .then(() => player.coordinator.play())
-                     .then(() => resolve('success'));
+                     .then(() => resolve());
                  } else {
                    reject('No match found for ' + name);
                  } 
                }
              });
            }
         });
-    }, Promise.resolve('success')); 
+    }, Promise.resolve()); 
   }
```

Comment:
You should also probably extract the login to a separate function that would return a promise, then you could make a much cleaner promise chain in playPandora. 

I can fix it though if I'm being unclear after merge.